ContractsApi.Tests/V1/Boundary/CreateContractRequestObjectValidatorTests.cs

```diff
@@ -0,0 +1,31 @@
+using ContractsApi.V1.Boundary.Requests;
+using ContractsApi.V1.Boundary.Requests.Validation;
+using AutoFixture;
+using FluentAssertions;
+using FluentValidation.TestHelper;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace ContractsApi.Tests.V1.Boundary.Request.Validation
+{
+#nullable enable
+    public class CreateContractRequestObjectValidatorTests
+    {
+        private readonly CreateContractRequestObjectValidator _ccrov;
+        private readonly Fixture _fixture = new();
+
+        public CreateContractRequestObjectValidatorTests()
+        {
+            _ccrov = new CreateContractRequestObjectValidator();
+        }
+
+        [Fact]
+        public void RequestShouldErrorForNoContractHierarchyd()
+        {
+            var model = new CreateContractRequestObject() { ContractManagement = null };
+            var result = _ccrov.TestValidate(model);
+            result.ShouldHaveValidationErrorFor(x => x.ContractManagement);
```

Comment:
We often check that the error message is correct as well. If adding multiple rules against the property (please see other comment re: validator) it's useful to check why exactly the validation failed, so you can test the logic properly for all scenarios.

Reply:
Addressed in [ae99290](https://github.com/LBHackney-IT/contracts-api/pull/91/commits/ae99290b0875239a32eb977d98d91d122305fbce)